customBoogie.patch

```diff
@@ -1,10 +1,10 @@
 diff --git a/Source/Dafny.sln b/Source/Dafny.sln
-index 36bb4d93b..0dea612cf 100644
+index 88200dcc4..3a75b0cc5 100644
 --- a/Source/Dafny.sln
 +++ b/Source/Dafny.sln
-@@ -30,6 +30,34 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DafnyTestGeneration", "Dafn
+@@ -34,6 +34,34 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoExtern", "AutoExtern\Au
```

Comment:
This change doesn't seem right. I think this patch should mostly just add entries.

Reply:
You're looking at a second-order diff (a diff of a patch); adding entries changes the context in which the patch applies, hence the change.

Reply:
Ah, right, of course.